servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/CompletableMergeWithPublisher.java

```diff
@@ -72,25 +66,20 @@ void handleSubscribe(final Subscriber<? super T> subscriber, final SignalOffload
         @Nullable
         private volatile TerminalSignal terminal;
 
-        MergerDelayError(Subscriber<? super T> subscriber, SignalOffloader signalOffloader, AsyncContextMap contextMap,
+        MergerDelayError(Subscriber<? super T> subscriber, AsyncContextMap contextMap,
                AsyncContextProvider contextProvider) {
-            // This is used only to deliver signals that originate from the mergeWith Publisher. Since, we need to
-            // preserve the threading semantics of the original Completable, we offload the subscriber so that we do not
-            // invoke it from the mergeWith Publisher Executor thread.
-            this.offloadedSubscriber = signalOffloader.offloadSubscriber(contextProvider.wrapPublisherSubscriber(
-                    subscriber, contextMap));
+            // This is used only to deliver signals that originate from the mergeWith Publisher.
+            this.offloadedSubscriber = contextProvider.wrapPublisherSubscriber(
```

Comment:
nit: `offloadedSubscriber` -> `wrapped`

also move previous line to end of current line.